### DISCUSSION TITLE
ModMenu GTA

### DISCUSSION THREAD

**Discussion by Ponthezs** [Ponthezs/ModMenuGta#1] [Q&A] [answered]
Mod Menu em jogos é um tipo de software modificado que é adicionado a um jogo para permitir que o jogador tenha acesso a recursos e funcionalidades extras que não estão disponíveis na versão original do jogo.

**Comment by PontesS2** [comment, marked as answer]
Sim está correto oque disse!

**Reply by Ponthezs** [reply]
Sim

**Comment by PontesS2** [comment]
Você Sabe me dizer sé e compativel com qualquer W10?

**Reply by Ponthezs** [reply]
Sim qulaquer um
